bin/node-sass

```diff
@@ -161,11 +163,17 @@ function getEmitter() {
   });
 
   emitter.on('log', function(data) {
-    console.log(data);
+    // If we need to outout large size buffer,
+    // keep waiting until buffer outputted completely.
+    asyncWaiting = true;
+
+    // In nodejs <= 7.1.0, console.log cannot write output larger than 8275 bytes data.
+    // So that, we output partial strings manually with some asynchronous way.
+    stdout(data);
   });
 
   emitter.on('done', function() {
-    if (!options.watch && !options.directory) {
+    if (!options.watch && !options.directory && !asyncWaiting) {
```

Comment:
`asyncWaiting` is initialized on false when called `getEmitter()`.
https://github.com/sass/node-sass/pull/1834/files#diff-c19ab2481e2703fcf0d2808f4c762f63R147

I read project source code, emit `log` event only stdout case ( is it right? ).
So that, `asyncWaiting` set to be true only emitted `log` event.

Please advise if there are other concerns that you need to worry about. thanks!